activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -126,6 +126,8 @@ def initialize(...)
 
         @config[:strict] = ConnectionAdapters::SQLite3Adapter.strict_strings_by_default unless @config.key?(:strict)
         @connection_parameters = @config.merge(database: @config[:database].to_s, results_as_hash: true)
+
```

Comment:
Nit: Let's ✂️ this line!
```suggestion
```

Reply:
Yep, I think that snuck in between some changes. Will remove.